openstack/swift/alerts/swift-api.alerts

```diff
@@ -32,7 +32,7 @@ groups:
       summary: swift-service-unavailable
 
   - alert: OpenstackSwiftHealthCheck
-    expr: avg(swift_health_statsd_exit_code) BY (region) > 0.2
+    expr: (avg(swift_recon_exit_code) BY (region) OR avg(swift_dispersion_exit_code) BY (region)) > 0.2
```

Comment:
The `or` might mask alarms here. `x or y` returns x if it is not 0. So if `avg(swift_recon_exit_code)` is above 0, but below 0.2, that value will be returned from the `... or ...` expression and the alert will not fire, even if `avg(svg_dispersion_exit_code)` was higher than 0.2.

Please make two separate alerts for swift-recon failing and swift-dispersion-report failing. Also, since we have the actual failed commandline in the labels for those metrics, that would be nice to have in the alert description, e.g. `swift-recon --unmounted has failed` instead of just `Swift health check failures`.